cscs-checks/apps/namd/namd_check.py

```diff
@@ -49,7 +49,7 @@ def __init__(self, arch, scale, variant):
 
         self.maintainers = ['CB', 'LM']
         self.tags = {'scs', 'external-resources'}
-        self.strict_check = False
+        self.strict_check = True
```

Comment:
You can remove this completely, since `True` is the default.

Reply:
Reference on Eiger is updated for the current module, which is for the SMP build of NAMD. According to the comments reported here https://jira.cscs.ch/browse/SD-51014, I need to investigate a bit more the NAMD affinity options in order to get better result (currently a pure MPI build provides better performances).